### PR TITLE
OCPBUGS-2057: Removing duplicate ref sentence

### PR DESCRIPTION
**Pull request by amolnar-gh**
Version(s): 4.11, 4.12

Issue: https://issues.redhat.com/browse/OCPBUGS-2057

Link to docs preview: https://51341--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-low-latency-tuning.html#cnf-understanding-workload-hints_cnf-master

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/cnf-understanding-workload-hints.adoc [hunk 1]
```diff
@@ -9,7 +9,7 @@
 The following table describes how combinations of power consumption and real-time settings impact on latency.
 [NOTE]
 ====
-The following workload hints can be configured manually. See "Creating a performance profile" for further information. You can also work with workload hints using the Performance Profile Creator. See "Creating a performance profile" for further information.
+The following workload hints can be configured manually. You can also work with workload hints using the Performance Profile Creator. See the "Creating a performance profile" section for further information.
```
Comment (aireilly) [inline modules/cnf-understanding-workload-hints.adoc:12]: ISG: https://www.ibm.com/docs/en/ibm-style?topic=references-citations "Use an italic font for the title unless the title is also a link, in which case use normal link highlighting."

```suggestion
The following workload hints can be configured manually. You can also work with workload hints using the Performance Profile Creator. See _Creating a performance profile_ for further information.
```

I would also suggest adding an `.Additional resources` section directly after in the assembly with a link to the section in question.

**Review by aireilly — COMMENTED** [review]
Follow ISG for internal references.

File: modules/cnf-understanding-workload-hints.adoc (hunk 1 above)
Comment (aireilly) [reply modules/cnf-understanding-workload-hints.adoc]: @amolnar-rh I stand corrected, when referring to another section of the same document, quotes are correct.

**Comment by amolnar-rh** [comment]
Please, cherry pick to 4.11 and 4.12.

Could you please also remove the `jira/invalid-bug` label? It disqualifies the PR from Hacktoberfest :)

**Pull request merged**